pkgs/top-level/all-packages.nix

```
@@ -9203,6 +9203,8 @@ in
 
   kati = callPackage ../development/tools/build-managers/kati { };
 
+  kcc = callPackage ../applications/graphics/kcc { };
```

```suggestion
  kcc = python3Packages.callPackage ../applications/graphics/kcc { };
```

Then you don't need to use `python3Packages` at all in the derivation and can take the python dependencies directly as argument.